README.md

```diff
@@ -44,54 +62,126 @@ This goes both ways: when explaining something make sure to present enough conte
+
+
+### Sprints
+
+Our sprints are 2 weeks long. We have the following meetings:
+
+- **Sprint Planning**: Individual teams hold sprint planning meetings at the start of the sprint. Work is estimated, tickets are assigned, and the team determines how much they can commit to completing. Everyone is expected to review potential upcoming work prior to sprint planning in order to estimate and discuss the work effectively.  
+- **Mega-groom**: The full team meets to discuss work for the upcoming sprint. This is scheduled the week before sprint planning. The meeting has an agenda that is both created by both product managers and tech leads. In addition to providing visibility into upcoming work cross-team, approach and open questions are discussed at a high level. Everyone should review this document prior to the meeting so they can come prepared to discuss the work.
+- **Sprint Demos**: Near the end of the sprint on Thursday afternoons, members volunteer to demo what they've worked on. This meeting involves the full team and optionally other department members. Demos should not be overly long or technical as the audience of demos involves non-technical folks. If a deep dive or longer technical demo is desired, we can use the following week's Dev Roundtable (see below about Dev Roundtables) slot.
+- **Sprint Retro**: Immediately following demos, we meet to discuss What Went Well, What to To Improve, and Other Discussion Topics. We hold an all-teams retro, which is great for general culture, surfacing larger questions or issues, and providing full team updates. We also typically celebrate a person who’s been a Goddamn Champion (GDC) in having a positive impact through their work over the past sprint. The GDC winner receives a $25 gift card.
+
+_**Another regular meeting to note:** We typically have a brown bag lunch meeting on Wednesdays at noon. It is called Dev Roundtable. These meetings are typically technical and can involve technical demos/presentations, working sessions, or discussions._
+
+Sprints should not be booked more than **75% full**. This gives buffer for sick time, unplanned work, work that was underestimated at the start of the sprint, and breathing room for individual professional development. Additionally, a minimum 3 points of sustaining engineering work should be tackled every sprint. If a team is experiencing sprint rollover, they should step back, and re-evaluate their sprint planning strategy. Sprint rollover should be the exception not the rule.
```

Comment:
FYI on Applications, we don't always point sustaining engineering work as they're often Chores or Bugs, this might be the same on other teams

Reply:
Should we be pointing them? We've had a lot of back and forth about this on voter registration with consensus starting to trend towards pointing that kind of work.

Reply:
Mmmmhh, I'm hesitant to because if you're consistently pulling in bugs and chores your velocity does level out and for me, it's a clear distinction between feature work and sustaining eng  -- maybe worth talking about at the next retro?

Reply:
I'll just leave this as so we have some kind of measurement to refer to for how much sustaining work to tackle each sprint

Reply:
I agree with Jen's reasoning!

One warning – once you decide to estimate bugs & chores, you can never undo that choice. I had to enable it for Platforms when we did the team merger, since at some point we'd flipped the switch on for Storm and so those cards could only exist on boards with that setting.

Reply:
I'd like to understand this one more. I admit that I'm the one who turned on points in the storm board as I needed a better understanding of how long things were taking and what was reasonable relative to other items. Is the concern that we'll push less features if we do this?

Reply:
The [idea](https://www.pivotaltracker.com/help/articles/planning_with_velocity/#bugs_and_chores) is basically that those types of tickets are overhead and (while still work) they decrease the "output" of the team. When we estimate them, a team that spent 100% of it's time spinning it's wheels on bugs and other busy-work may look super-productive.

I don't think it's a big deal either way, and I don't think we put much stock in the story-pointed-based "team velocity" anyways. But it's interesting food for thought!